### PR TITLE
Code style cleanup.

### PR DESCRIPTION
**Pull request by davebx**

### CODE REVIEW

File: lib/tool_shed/utility_containers/utility_container_manager.py [hunk 1]
```diff
@@ -835,10 +825,10 @@ def key_is_current_repositorys_key( self, repository_name, repository_owner, cha
...
             key_only_if_compiling_contained_td = components_list[ 5 ]
         if repository_name == key_name and \
-            repository_owner == key_owner and \
-            changeset_revision == key_changeset_revision and \
-            prior_installation_required == key_prior_installation_required and \
-            only_if_compiling_contained_td == key_only_if_compiling_contained_td:
+                repository_owner == key_owner and \
+                changeset_revision == key_changeset_revision and \
+                prior_installation_required == key_prior_installation_required and \
+                only_if_compiling_contained_td == key_only_if_compiling_contained_td:
```
Comment (dannon) [inline lib/tool_shed/utility_containers/utility_container_manager.py]: Generally speaking, slash continuations should be avoided wherever possible.  Here, I'd say:

```
         if ( repository_name == key_name and
              repository_owner == key_owner and
              changeset_revision == key_changeset_revision and
              prior_installation_required == key_prior_installation_required and
              only_if_compiling_contained_td == key_only_if_compiling_contained_td ):
```

Comment (davebx) [inline lib/tool_shed/utility_containers/utility_container_manager.py]: I wasn't sure of the best practice for this. Now I am.

Comment (dannon) [inline lib/tool_shed/utility_containers/utility_container_manager.py]: Hooray!  :)

**Comment by dannon** [comment]
@davebx Thanks for the fix!  I'd generally group the other multiple assignments the same way to avoid slash continuation there too, but that can wait until the namedtuple overhaul.

**Pull request merged**